types/tsconfig-augment-codec.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "./tsconfig-base.json",
+  "compilerOptions": {
+    "noEmit": true, // No need to actually create any output
```

Comment:
is the json file correctly parsed despite the non-standard comments in json ?

Reply:
Yes, comments seem to be allowed in `tsconfig.json` (related PR: https://github.com/microsoft/TypeScript/pull/5450) even though it's technically not a valid json.

Reply:
This file is actually used during the CI checks now (it's part of `check:augment` script which is part of `checks`)